### PR TITLE
[SDK-4499] Update to use the app router

### PR DESCRIPTION
**Pull request by adamjmcgrath**
See also https://github.com/auth0/docs/pull/10293

### CODE REVIEW

File: Sample-01/tests/pages/api/shows.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * @jest-environment node
```
Comment (Widcket) [inline Sample-01/tests/pages/api/shows.test.js:2]: Should we update the dir structure of the tests?

**Review by Widcket — COMMENTED** [review]
With these changes, the sample app will no longer serve as an example of Next.js SDK + pages router, only with the app router, right?
Should we not showcase both, given that both are supported, and are likely being used in many real world apps due to incremental migration to the app directory?

**Comment by Widcket** [comment]
Even the Next.js documentation features them both:

Screenshot 2023-08-24 at 11 50 50 PM

File: Sample-01/tests/pages/api/shows.test.js (hunk 1 above)
Comment (adamjmcgrath) [reply Sample-01/tests/pages/api/shows.test.js:2]: Done [e2220b5](https://github.com/auth0-samples/auth0-nextjs-samples/pull/131/commits/e2220b51cb725f3459b2e601906a677500d9a92e)

**Comment by adamjmcgrath** [comment]
> With these changes, the sample app will no longer serve as an example of Next.js SDK + pages router, only with the app router, right?

Correct

> Should we not showcase both, given that both are supported, and are likely being used in many real world apps due to incremental migration to the app directory?

I disagree, there are many use cases not covered in this sample - including incremental migration. Adding support for pages would add too much bloat and would not represent a simple sample app. (we don't for example cover middleware in the sample either). For the swiss-army knife, cover all example - we have [the example app in the repo](https://github.com/auth0/nextjs-auth0/tree/main/example-app)

**Review by Widcket — APPROVED** [review]

**Comment by Widcket** [comment]
BTW, I don't think I mentioned that the sample app needs to cover all use cases 🤔 I wonder what gave you that impression.

**Pull request merged**